nixos-modules/microvm/mounts.nix

```diff
@@ -110,7 +110,7 @@ lib.mkIf config.microvm.guest.enable {
       }) {} (withDriveLetters config.microvm)
   ) (
     # 9p/virtiofs Shares
```

Comment:
shares != volumes

Is there a use-case for not mounted 9p/virtiofs?

Reply:
I am unsure of the semantics between shares & volumes here ?

This PR allows me to mount my ZFS zpool inside the guest, without expecting a filesystem. 
I set this config in the guest machine configuration:
```
microvm = {
  volumes = [
    {
      mountPoint = null;
      image = "/dev/disk/by-label/my-zpool";
      size = 1024;
    }
  ];
};
```

My use case is that I have a service expecting root access to a whole ZFS zpool in the VM.

Reply:
- volumes: Block devices
- shares: Directories

A ZVOL should be use as a volume. It already allows for `mountPoint = null`.

Reply:
Does that distinction clarify your issue?

Reply:
It seems it was supported all along and I wrote an erroneous configuration the first time around, giving me wrong assumptions.
PEBCAK issue.

Thank you for taking the time, appreciated